client/menu.cpp

```diff
@@ -2899,10 +2899,14 @@ void mr_menu::back_to_menu()
 
   if (is_server_running()) {
     ask = new hud_message_box(king()->central_wdg);
-    ask->set_text_title(_("Leaving a local game will end it!"),
-                        QStringLiteral("Leave game"));
-    ask->setStandardButtons(QMessageBox::Cancel | QMessageBox::Ok);
-    ask->setDefaultButton(QMessageBox::Cancel);
+    ask->set_text_title(
+        _("Do you want to leave the game?\n\n Leaving a single-player game "
```

Comment:
I think the space after \n will result in a line slightly off center

Reply:
This is what it looks like on my system
![image](https://user-images.githubusercontent.com/23037938/190926251-5081e300-ce7b-4f1f-9049-793aafe25ba5.png)

Reply:
Second line: 25px on the left, 20px on the right

Reply:
Yep, it sure does. I'll push another commit